src/pages/membership.tsx

```diff
@@ -84,7 +84,7 @@ function MembershipPage(props: Props) {
                             </Text>
                         </li>
                         <li>
-                            <Text>Free Event Tickets to Tour of Taiwan and Night Market</Text>
+                            <Text>Free Event Tickets to Night Market</Text>
```

Comment:
can we add a line that says free entry to tour of taiwan, also update that in the section that says "what is a tasa membership card?"

Reply:
we aren't doing free entry to tour of taiwan anymore. Only free raffles (decided at today's board meeting)

Reply:
gonna submit another pull request